### PR TITLE
Set job node state in error for Ansible job when failure occurs

### PR DESCRIPTION
**Pull request by stebenoist**
# Pull Request description

### How to verify it
See issue description
### Description for the changelog
job node state remains to "executing" when Ansible job fails ([GH-455](https://github.com/ystia/yorc/issues/455))
## Applicable Issues
#455

### CODE REVIEW

File: prov/ansible/monitoring_job.go [hunk 1]
```diff
@@ -78,6 +80,7 @@ func (o *actionOperator) ExecAction(ctx context.Context, cfg config.Configuratio
 			if opErr == nil {
 				opErr = errors.Errorf("job implementation of node %q was detected as failed", nodeName)
 			}
+			deployments.SetInstanceStateStringWithContextualLogs(ctx, consulutil.GetKV(), deploymentID, action.Data["nodeName"], "0", tosca.NodeStateError.String())
```
Comment (laurentganne) [inline prov/ansible/monitoring_job.go]: No need to call deployments.SetInstanceStateStringWithContextualLogs() when the state is completed ?

Comment (stebenoist) [reply prov/ansible/monitoring_job.go]: Yes, I think you're right

Comment (stebenoist) [reply prov/ansible/monitoring_job.go]: But which state ? OK or "completed" ?

Comment (laurentganne) [reply prov/ansible/monitoring_job.go]: It seems for Slurm Jobs, this is `Completed`, while for kubernetes jobs this is `Succeeded` and the failure is `Failed`

Comment (stebenoist) [reply prov/ansible/monitoring_job.go]: Ok it seems to be the specific states for job schedulers. But in our case, there is no job scheduler and we should just follow Tosca nodes states ? WYT ?

Comment (laurentganne) [reply prov/ansible/monitoring_job.go]: ok, started then ?

Comment (stebenoist) [reply prov/ansible/monitoring_job.go]: Yes

Comment (stebenoist) [reply prov/ansible/monitoring_job.go]: Done

**Pull request merged**